Makefile

```diff
@@ -1,6 +1,10 @@
 release: clean
+	git checkout master
+	git pull origin master
 	python setup.py sdist bdist_wheel
 	python -m twine upload dist/*
+	git tag `python -c "print(__import__('sphinx_search').__version__)"`
+	git push --tags
```

Comment:
Not sure if this is too much automation, let me know what you think

Reply:
Yea, I don't think we want to autotag & push, I could see issues there with not having incremented the version. Maybe have `tag` be another task?